src/Microsoft.Azure.SignalR.Common/ClientInvocation/IClientResultsManager.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NET7_0_OR_GREATER
+using System;
+using Microsoft.Azure.SignalR.Protocol;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Buffers;
+using Microsoft.AspNetCore.SignalR.Protocol;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface IClientResultsManager
+    {
+        public ulong GetNewInvocation();
```

Comment:
why invocationId here is ulong while below invocationId is string

Reply:
I wrote a clearer method to generate invocation id. This old `ulong GetNewInvocation` is actually only a part of actual InvocationId (the 3rd part). 
```
public string GetNewInvocationId(string connectionId, string serverGUID)
{
  return $"{connectionId}-{serverGUID}-{Interlocked.Increment(ref _lastInvocationId)}";
}
```